packages/kbn-grid-layout/grid/grid_panel.tsx

```diff
@@ -129,6 +131,63 @@ export const GridPanel = forwardRef<
       []
     );
 
+    useEffect(
+      () => {
+        /** Update the styles of the panel via a subscription to prevent re-renders */
+        const styleSubscription = combineLatest([
+          gridLayoutStateManager.gridLayout$,
+          gridLayoutStateManager.expandedPanelId$,
+          gridLayoutStateManager.isMobileView$,
```

Comment:
Are `expandedPanelId$` and `isMobileView$` related? Could these be seperate subscriptions?

Reply:
yes, good catch! https://github.com/elastic/kibana/pull/200771/commits/53ee088facd4e2ca5fbb922f9b7e1bd697e9d4aa